flake.nix

```diff
@@ -45,12 +49,14 @@
             packages = [
               pkgs.applyconfiguration-gen
               pkgs.controller-gen
+              pkgs.chart-testing
```

Comment:
There's a test in `operator/pkg/lint_test.go` that checks the versions of various tools with a golden file. It's there so we can easily update nixpkgs and see any affected versions of tools we use. Could you add helm-docs and chart-testing to it?

Reply:
Yes I will

Reply:
We have now 2 tests 😮‍💨 `pkg/lint_test.go` and `operator/pkg/lint_test.go` I will combine them.